besecure_developer_toolkit/src/create_ossp_master.py

```diff
@@ -17,73 +18,76 @@ def __init__(self, issue_id: int, name: str) -> None:
         self.issue_id = issue_id
         self.name = name
 
-    def check_issue_exists(self, issue_id) -> None:
-        """
-            Check for issue id is exits or not.
+    @staticmethod
+    def check_issue_exists(issue_id: int):
+        """Function to check if the issue exists
+
+        Args:
+            issue_id (int): Issue issue_id
+
+        Returns:
+            bool: True if issue exists and False if otherwise
         """
+        url = f"https://github.com/Be-Secure/Be-Secure/issues/{issue_id}"
         try:
-            urlopen('https://github.com/Be-Secure/Be-Secure/issues/'+str(issue_id))
-        except Exception as err:
-            exc_type, exc_obj, exc_tb = sys.exc_info()
-            fname = os.path.split(exc_tb.tb_frame.f_code.co_filename)[1]
-            print(exc_type, fname, exc_tb.tb_lineno)
-            print(
-                f"Could not find issue with id: {str(issue_id)}, error: {str(err)}")
-            sys.exit()
+            response = requests.head(url, timeout=10)
+            return response.status_code < 400
+        except requests.exceptions.RequestException:
+            return False
 
     def check_issue_related_to_project(self):
         """
             Check project name from issue
         """
-        json_data = json.loads(urlopen(
-            f'https://api.github.com/repos/Be-Secure/Be-Secure/issues/{self.issue_id}').read())
-        issue_title = json_data["title"]
-        project_name = str(str(issue_title).split(":")[1]).replace(" ", "")
-        if project_name != self.name:
-            print(
-                f"[bold red]Alert! [yellow]Mismatch issue_id-project : [green] \
-                    Issue id {self.issue_id} does not match the project {self.name}")
-            sys.exit()
+        url = f'https://api.github.com/repos/Be-Secure/Be-Secure/issues/{self.issue_id}'
+        with urllib.request.urlopen(url) as raw_data:
+            json_data = json.loads(raw_data.read())
+            issue_title = json_data["title"]
+            project_name = str(str(issue_title).split(":")[1]).replace(" ", "")
+            if project_name != self.name:
+                print(
+                    f"[bold red]Alert! [yellow]Mismatch issue_id-project : [green] \
```

Comment:
can you fix this string as I am getting lots of space in terminal

```
Enter OSSP name:owlviewer
Alert! Mismatch issue_id-project :                          Issue id 89 does not match the project owlviewer
```

Reply:
@sudhirverma Done